src/Tizen.NUI/src/public/XamlBinding/Transition.cs

```diff
@@ -100,6 +101,45 @@ public int EndTime
         }
     }
 
+
+    /// <summary>
+    /// It is the container to contain the behaviors of Transition.
+    /// </summary>
+    /// This will be public opened in tizen_6.5 after ACR done. Before ACR, need to be hidden as inhouse API.
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class BehaviorContainer : List<AnimationBehavior>
+    {
+        private Dictionary<string, AnimationBehavior> behaviors = new Dictionary<string, AnimationBehavior>();
+
+        /// <summary>
+        /// The method for user to add behavior.
+        /// </summary>
+        /// This will be public opened in tizen_6.5 after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void Add(object obj)
+        {
+            var behavior = obj as AnimationBehavior;
+
+            if (null != behavior)
+            {
+                behaviors.Add(behavior.Key, behavior);
+            }
+        }
+
+        /// <summary>
+        /// The method for user to get the behavior by the key.
+        /// </summary>
+        /// This will be public opened in tizen_6.5 after ACR done. Before ACR, need to be hidden as inhouse API.
```

Comment:
In TCSACR-395, I couldn't see any "Add-ACR". so I understand that this is added as Hidden-API, then please **REMOVE** this comment. : `/// This will be public opened in tizen_next after ACR done. Before ACR, need to be hidden as inhouse API.`
thank you.